src/app/fdctl/run.c

```diff
@@ -211,6 +217,8 @@ solana_labs_main( void * args ) {
   ADD1( "fdctl" );
   ADD( "--log", "-" );
 
+  ADD( "--firedancer-pod", "" );
```

Comment:
This appears to cause `fddev` to crash with:
```
error: Found argument '--firedancer-pod' which wasn't expected, or isn't valid in this context

USAGE:
    fdctl --ledger <DIR> --log <FILE>

For more information try --help
```

Reply:
Ah :) stray line from another change.